src/asmcnc/skavaUI/screen_go.py

```diff
@@ -691,13 +688,33 @@ def start_or_pause_button_press(self):
 
                 self.m.s.is_ready_to_assess_spindle_for_shutdown = True # allow spindle overload assessment to resume
         else:
-            self._start_running_job()
-            self.jd.job_start_time = time.time()
+            has_health_check_run = self.m.has_spindle_health_check_passed() or self.m.has_spindle_health_check_failed()
+
+            self.m._grbl_soft_reset()
```

Comment:
don't think this is needed - health check also has a soft reset, and this doesn't need to run if user if not on + or is not using health check